docs/07. media library/03. integrating in your module.md

```diff
@@ -115,7 +116,7 @@ Just pass the media group entity to your custom entity constructor.
 
 ### In your Update Command class
 
-```php
+``php
```

Comment:
you accidentally removed a backtick here

Reply:
Fixed it